clang/lib/CodeGen/CodeGenAction.cpp

```diff
@@ -257,37 +281,19 @@ bool BackendConsumer::LinkInModules(llvm::Module *M, bool ShouldLinkFiles) {
     CurLinkModule = LM.Module.get();
     bool Err;
```

Comment:
Got error:
```
clang/lib/CodeGen/CodeGenAction.cpp:282:10: error: variable 'Err' set but not used [-Werror,-Wunused-but-set-variable]
    bool Err;
         ^
1 error generated.
```

Reply:
Oversight on my part. Should be fixed by https://github.com/llvm/llvm-project/pull/81777

Reply:
Thanks!